packages/ember-metal/lib/streams/key-stream.js

```diff
@@ -25,6 +25,12 @@ let KeyStream = BasicStream.extend({
 
   compute() {
     var object = this.sourceDep.getValue();
+    const type = typeof object;
+
+    if (type === 'boolean' || type === 'string') {
```

Comment:
@krisselden thats not possible, Chrome is the fastest runtime in the world :trollface: